beets/importer.py

```diff
@@ -943,7 +943,9 @@ def is_archive(cls, path):
         if not os.path.isfile(path):
             return False
 
-        for path_test, _ in cls.handlers():
+        for path_test, x in cls.handlers():
+            if isinstance(path, bytes):
+                path = util.py3_path(path)
```

Comment:
i assume we're seeing bytes here because the results from TarFile and friends are strings